webrtc.html

```diff
@@ -528,23 +520,60 @@
           </li>
 
           <li>
-            <p>Create a <code><a>MediaStream</a></code> object
-            <var>stream</var>, to represent the incoming media stream.</p>
-          </li>
+            <p>Run the following steps to create a track
+            representing the incoming component:</p>
 
-          <li>
-            <p>Run the <a href="#represent-component-with-track">algorithm</a>
-            to represent an incoming component with a track for each incoming
-            component.</p>
+            <ol>
+              <li>
+                <p>Create a <code><a>MediaStreamTrack</a></code> object
+                <var>track</var> to represent the component.</p>
+              </li>
+
+              <li>
+                <p>Initialize <var>track's</var> <code><a href=
+                "getusermedia.html#dom-mediastreamtrack-kind">kind</a></code>
+                attribute to "<code>audio</code>" or "<code>video</code>"
+                depending on the media type of the incoming component.</p>
+              </li>
+
+              <li>
+                <p>Initialize <var>track's</var> <code><a href=
+                "getusermedia.html#dom-mediastreamtrack-id">id</a></code>
+                attribute to the component track id.</p>
```

Comment:
If this is MID, we can get it from media. If it is MSID, it needs to come from the remote description. I think there is a decent case for it being MID, but since this is also not new, I think we should deal with it later (e.g. when we get the .mid pull request).